syft/frameworks/torch/tensors/interpreters/additive_shared.py

```diff
@@ -462,6 +451,35 @@ def __getitem__(self, indices):
 
     ## SECTION SPDZ
 
+    def _basic_arithmetic_op(self, op, shares: dict, operand):
+        """Do specific operation(op) operand to the self AST instace.
+
+        Agrs:
+            op: a function to be applied for self AST instance and operand.
+            shares: a dictionary <location_id -> PointerTensor) of shares corresponding to
+                self. Equivalent to calling self.child.
+            other: the operand being added to self, can be:
+                - a dictionary <location_id -> PointerTensor) of shares
+                - a torch tensor
+                - a constant
+
+        """
+        if isinstance(operand, int):
+            operand = torch.tensor([operand], dtype=self.torch_dtype)
+
+        if isinstance(operand, (torch.LongTensor, torch.IntTensor)):
+            operand = operand.share(
+                *self.child.keys(), **self.get_class_attributes(), **no_wrap
+            ).child
+        elif not isinstance(operand, dict):
```

Comment:
Q: Couldn't this ```elif``` be moved upper? (like to be an ```elif``` for the upper ```if```)

Reply:
Maybe we couldn't. As I changed it to what you suggested, it does not pass the test. 😄

Reply:
Do you still have the stacktrace at hand (maybe send me a message on slack)

Reply:
I sent you a message!

Reply:
Ahh..I see...I read this like ```elif isinstace``` without the ```not```